apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+            '<div id="player-cover-album" data-l10n-id="songAlbum">Song Album</div>' +
+            '<button id="player-cover-share" class="player-cover-button" data-icon="share" data-l10n-id="share-song"></button>' +
+          '</div>' +
+          '<div id="player-cover-buttons">' +
+            '<button id="player-album-repeat" class="player-cover-button" data-icon="repeat"></button>' +
+            '<div id="player-album-rating" data-l10n-id="player-album-rating" role="radiogroup">' +
+              '<button class="rating-star" data-rating="1" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"1"}\'></button>' +
+              '<button class="rating-star" data-rating="2" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"2"}\'></button>' +
+              '<button class="rating-star" data-rating="3" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"3"}\'></button>' +
+              '<button class="rating-star" data-rating="4" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"4"}\'></button>' +
+              '<button class="rating-star" data-rating="5" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"5"}\'></button>' +
+            '</div>' +
+            '<button id="player-album-shuffle" class="player-cover-button" data-icon="shuffle"></button>' +
+          '</div>' +
+          '<section id="info-banner" role="status">' +
+            '<p data-l10n-id="music-paused-notification"></p>' +
+          '</section>' +
+        '</div>' +
+       '<div id="player-seek">' +
+          '<span id="player-seek-elapsed" dir="ltr">00:00</span>' +
+          '<div id="player-seek-bar">' +
+            '<progress id="player-seek-bar-progress" value="0"></progress>' +
+            '<div id="player-seek-bar-indicator"></div>' +
+          '</div>' +
+          '<span id="player-seek-remaining" dir="ltr">00:00</span>' +
+        '</div>' +
+        '<div id="player-controls" dir="ltr">' +
+          '<button id="player-controls-previous" class="player-controls-button" data-icon="skip-back"></button>' +
+          '<button id="player-controls-play" class="player-controls-button is-pause" data-icon="pause"></button>' +
+          '<button id="player-controls-next" class="player-controls-button" data-icon="skip-forward"></button>' +
+        '</div>' +
+        '<audio id="player-audio" class="hidden"></audio>' +
+      '</div>';
+        
+        //Insert the star-rating bar into the dom
+        document.body.insertAdjacentHTML('beforeend', starMarkUp);
+        
+        //Initialize the Player View
+        pv = PlayerView;
+
+        //Override #setSeekBar to avoid excess work in init()
+        pv.setSeekBar = function(){return 0;};
+
+        pv.init();
+        ratings = pv.ratings;
+    });
+    
+    suite('#setRating', function () {
+        
+        test("should set aria-checked to false on all stars", function () {
```

Comment:
The tests test the right thing, however we can clean it up a bit and avoid repetition. You can define a function around where pv and ratings are declared that will look like this:
```
function testRatingsAriaChecked(checkedIndex) {
  pv.setRatings(checkedIndex);
  Array.prototype.forEach.call(ratings, function(rating, index) {
    assert.equal(checkedIndex - 1 === index ? 'true' : 'false',
      rating.getAttribute("aria-checked"));
    });
}
```
We basically set the ratings based on the variable and then check that the star with the right index has the attribute set in the loop.
At this point your test is reduced to calling testRatingsAriaChecked with 0, 2, 4 and 5.